src/Stryker.Core/Stryker.Core/DashboardCompare/GitInfoProvider.cs

```diff
@@ -5,115 +5,123 @@
 
 namespace Stryker.Core.DashboardCompare
 {
+    using System.Linq;
+
+    using Logging;
+
+    using Microsoft.Extensions.Logging;
+
     public class GitInfoProvider : IGitInfoProvider
     {
+        private enum GitSourceKinds
+        {
+            Unknown,
+            Commit,
+            CanonicalBranchName,
+            FriendlyBranchName
+        }
+        
         private readonly StrykerOptions _options;
         private readonly string _repositoryPath;
+        private readonly ILogger<GitInfoProvider> _logger;
         public IRepository Repository { get; }
 
-        public string RepositoryPath
-        {
-            get
-            {
-                return _repositoryPath ?? LibGit2Sharp.Repository.Discover(_options.BasePath)?.Split(".git")[0];
-            }
-        }
+        public string RepositoryPath => this._repositoryPath ?? LibGit2Sharp.Repository.Discover(this._options.BasePath)?.Split(".git")[0];
```

Comment:
It seems you have some auto-formatter that adds this everywhere. We don't use this style in stryker.

Reply:
indeen rider did it il will revert

Reply:
done

Reply:
Could you fix it in the rest of the file as well

Reply:
already done i did replace all in the file

Reply:
Ah I must have been looking at an old version